nx-X11/programs/Xserver/hw/nxagent/Composite.c

```diff
@@ -24,18 +24,8 @@
 #include "Options.h"
 #include "Windows.h"
 
-#undef  NXAGENT_UPGRADE
-
-#ifndef NXAGENT_UPGRADE
-
-#include "NXcomposite.h"
```

Comment:
> So the old code does basically only ever #include "NXcomposite.h".

It does include NXcomposite.h always. That is correct, indeed. But if you look at NXcomposite.h, everything in it is embraced by 

```
#ifndef NXAGENT_UPGRADE
[...]
#endif /* #ifndef NXAGENT_UPGRADE */
´´´

So, we can also not include it right away and remove NXcomposite.h.